Mathlib/AlgebraicGeometry/EllipticCurve/Projective.lean

```diff
@@ -235,10 +239,10 @@ lemma equation_iff (P : Fin 3 → R) : W'.Equation P ↔
   rw [Equation, eval_polynomial, sub_eq_zero]
 
 lemma equation_smul (P : Fin 3 → R) {u : R} (hu : IsUnit u) : W'.Equation (u • P) ↔ W'.Equation P :=
-  have (u : R) {P : Fin 3 → R} (hP : W'.Equation P) : W'.Equation <| u • P := by
+  have hP (u : R) {P : Fin 3 → R} (hP : W'.Equation P) : W'.Equation <| u • P := by
```

Comment:
There are now two `hP`s. Isn't that a bit confusing? Or is it intentional?

Reply:
Probably fine? The first `hP` doesn't appear inside the `have`, and the second `hP` doesn't appear outside the `have`.